tests/unit/test_message_get.py

```diff
@@ -24,3 +38,11 @@ def test_get_msg_to_returns_newest_internal_user_if_known(self):
         to = get_msg_to(conversation)
 
         self.assertEqual(to, ['user2'])
+
+    @requests_mock.mock()
+    def test_get_msg_without_survey_name(self, mock_request):
+        mock_request.get(url_get_thread, json={'messages': [message_json], 'is_closed': False})
+        mock_request.get(url_get_conversation_count, json={'total': 0})
+        mock_request.get(url_get_survey_long_name, json={"longName": "None",})
+        response = self.app.get("secure-message/threads/9e3465c0-9172-4974-a7d1-3a01592d1594")
+        self.assertTrue('None'.encode() in response.data)
```

Comment:
This is a bit of a weak test, 'None' could easily appear anywhere in the HTML (in the business name for example).  Can you amend this to make it more likely that it's definitely looking at the survey name part?  It's okay to put a bigger block of HTML in if you need to

Reply:
Also I thought the template would render "No Business" or "No Survey" rather than None? Should we be checking for that?

Reply:
Quite right: fixed.